geode-redis/src/main/java/org/apache/geode/redis/internal/RegionProvider.java

```diff
@@ -51,4 +65,23 @@ public RegionProvider(InternalCache cache) {
   public Region<String, Object> getConfigRegion() {
     return configRegion;
   }
+
+  /**
+   * Validates that the value passed in is a power of 2 and that it is not greater than
+   * {@link #REDIS_SLOTS}
+   *
+   * @throws ManagementException if there is a problem with the value
+   */
+  protected static void validateBuckets(int buckets) {
+    if (buckets <= 0 || ((buckets & (buckets - 1)) != 0)) {
+      throw new ManagementException(
+          "Could not start Redis Server - redis region buckets must be a power of 2. Configured value is invalid: "
```

Comment:
Put the system property name "redis.region.buckets" in a constant and use it in the exception strings instead of "redis region buckets" to help communicate that they set the system property to an illegal value. Some readers of this exception may not be able to tie it back to the system property otherwise.

Reply:
Done